.claude/scripts/poll-pr.sh

```
@@ -1,191 +1,597 @@
+      echo ""
+    fi
+    exit_code=2
+
+  elif [ "$hr_count" -gt 0 ] &&        [ "$(echo "$human_reviews_json" | jq '[to_entries[] | select(.value == "CHANGES_REQUESTED")] | length')" -gt 0 ]; then
+    echo "  ACTION REQUIRED: Reviewer(s) requested changes"
+    echo "$human_reviews_json" | jq -r 'to_entries[] | select(.value == "CHANGES_REQUESTED") | "    - \(.key)"' 2>/dev/null || true
+    exit_code=3
+
+  elif [ "$hr_count" -eq 0 ] || [ "$hr_approved" -eq 0 ]; then
+    echo "  WAITING: Awaiting human review approval"
+    if [ -z "$prev_review_notified_at" ] && ( [ "${#REVIEWERS[@]}" -gt 0 ] || [ "${#TEAM_REVIEWERS[@]}" -gt 0 ] ); then
```

Shellcheck (0.11.0)

[style] Use { ..; } instead of (..) to avoid subshell overhead.

(SC2235)